jsonrpc/types.go

```diff
@@ -17,6 +17,8 @@ type transactionOrHash interface {
 type transaction struct {
```

Comment:
Can you add a test to make sure the encoding is correct? Shouldn't the `gasTipCap` and `gasFeeCap` be optional?

Reply:
Done

Reply:
Are `gasTipCap` and `gasFeeCap` optional?

Reply:
Yes

Reply:
They are not optional right now.